### ISSUE TITLE
Distracting repositioning when searching blocks

### ISSUE THREAD

**Issue by iseulde** [WordPress/gutenberg#998]
When you search for blocks in the "inserter", the whole box continuously resizes, which I find very distracting, and it's hard to follow what you're typing.

**Comment by mtias** [comment]
I think this is due to the search being at the bottom in the top inserter. We should move it to the top when the popover comes from the bottom.

**Comment by mtias** [comment]
Closing as we have a few tasks for the inserter, which includes showing search close to the button.
